src/python/gem5/resources/downloader.py

```diff
@@ -181,6 +172,27 @@ def _download(url: str, download_to: str, max_attempts: int = 6) -> None:
             )
 
 
+def get_proxy_context():
+    use_proxy = os.getenv("GEM5_USE_PROXY")
+    if use_proxy:
+        import socket
+        import ssl
+
+        import socks
+
+        IP_ADDR, host_port = use_proxy.split(":")
+        PORT = int(host_port)
```

Comment:
Why are these all caps?

Reply:
They were caps in the original code, I will change them to lower case.